src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -37,6 +37,19 @@ tnsr::aa<DataType, Dim, Frame> spacetime_metric(
   return spacetime_metric;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::ii<DataType, SpatialDim, Frame> spatial_metric(
+    const tnsr::aa<DataType, SpatialDim, Frame>& spacetime_metric) noexcept {
+  auto spatial_metric = make_with_value<tnsr::ii<DataType, SpatialDim, Frame>>(
+      spacetime_metric, 0.);
```

Comment:
Is `make_with_value(..., 0)` the best initialization here? All these tensor components are overwritten in the for loops below, so setting an initial value is unnecessary work.

Maybe I am worrying about a premature optimization.